### PR TITLE
Pv2 3209 upgrade net6

### PR DESCRIPTION
**Pull request by David-B-Read**

### CODE REVIEW

File: src/SFA.DAS.Payments.AnonymiserTool/Program.cs [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
 {
     static class Program
     {
-        public static string FileLocationKey = "DcToolOutputDirectory";
+        public static string FileLocationKey = "Settings:DcToolOutputDirectory";
 
         static async Task Main(string[] args)
         {
+            ConfigurationHelper.Initialize(new ConfigurationBuilder()
+                .AddJsonFile($"appsettings.json")
```
Comment (LiamWright) [inline src/SFA.DAS.Payments.AnonymiserTool/Program.cs:20]: Do we need an additional check for existence of a appsettings.development.json file?

File: src/SFA.DAS.Payments.Contingency/CongingencyStrategies/R13Payments.cs [hunk 2]
```diff
@@ -21,7 +22,7 @@ public async Task GenerateContingencyPayments()
             Console.WriteLine("Processing R13 Payments...");
 
             // Load data
-            using (var connection = new SqlConnection(ConfigurationManager.ConnectionStrings["DASPayments"].ConnectionString))
+            using (var connection = new SqlConnection(ConfigurationHelper.Configuration.GetConnectionString("DASPayments")))
```
Comment (LiamWright) [inline src/SFA.DAS.Payments.Contingency/CongingencyStrategies/R13Payments.cs:25]: Is it worth writing a class around the config to access the values via variable? Similar to the TestsConfiguration? https://github.com/SkillsFundingAgency/das-payments-v2-acceptancetests-common/blob/06cb65f9176d059e76a6c10ddb4e449457a082dc/src/SFA.DAS.Payments.AcceptanceTests.Core/Infrastructure/TestsConfiguration.cs

Comment (David-B-Read) [reply src/SFA.DAS.Payments.Contingency/CongingencyStrategies/R13Payments.cs:25]: The configuration is different for each console app, so we'd likely need a configuration class for each. This just exposes the inbuilt .net IConfiguration interface

File: src/SFA.DAS.Payments.AnonymiserTool/Program.cs (hunk 1 above)
Comment (David-B-Read) [reply src/SFA.DAS.Payments.AnonymiserTool/Program.cs:20]: AFAIK these are tools that are run by dev/tests on their local machine

**Review by LiamWright — APPROVED** [review]

**Comment by David-B-Read** [comment]
Closing PR as it was agreed that these tools are from the V1 to V2 upgrade and are no longer required. The FM36 Tool has been moved to the [das-payments-v2-test-tools](https://github.com/SkillsFundingAgency/das-payments-v2-testtools) repo

**Pull request closed without merging**